### PR TITLE
#92 Login History(activities)

### PR DESCRIPTION
**Pull request by pacifiquemboni**
### What does this PR do?
[x] Allowing trainee to see Login History(Activities)
[x] Translating attendance

### How should this be manually tested?
To test this you need to clone this project and switch to `ft-login-activies` branch, install all dependencies `npm install` then start the project `npm start`.
after successfull logging in click on profile icon then choose `login activity` and ther you are

### CODE REVIEW

**Review by MugemaneBertin2001 — APPROVED** [review]

**Review by munezeromicha — APPROVED** [review]

**Review by ncutixavier — COMMENTED** [review]
- No translation
- File naming is wrong

**Comment by pacifiquemboni** [comment]
@ncutixavier  i have worked on comments given

**Review by ncutixavier — COMMENTED** [review]
For the naming convention, follow CamelCase
image

**Comment by ncutixavier** [comment]
@pacifiquemboni  I can see reason why you're changing the package.json

File: package.json
Comment (ncutixavier) [inline package.json:1]: Why are you updating packages?

Comment (pacifiquemboni) [reply package.json:1]: to remove warning that show up when you start the app
WhatsApp Image 2024-11-10 at 17 47 42_09a9cb68

Comment (pacifiquemboni) [reply package.json:1]: here is the results after updating packages
WhatsApp Image 2024-11-10 at 23 36 24_0a90618a

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**